enrichments/trace/internal/elastic/span.go

```diff
@@ -200,6 +201,10 @@ func (s *spanEnrichmentContext) Enrich(
 			s.userAgentName = v.Str()
 		case semconv27.AttributeUserAgentVersion:
 			s.userAgentVersion = v.Str()
+		case "type":
+			if v.Str() == "mobile" {
```

Comment:
@lahsivjar pointed out on Slack that APM Server sets `transaction.type` always, when attribute `type` is present to that value. Seems to be done here: https://github.com/elastic/apm-data/blob/main/input/otlp/traces.go#L452-L453

I understand we want to solve the `transaction.type` issue for mobile. But to be aligned with APM Server, I'd propose we do the same as apm-data: always take the value from the attribute `type` and set that to `transaction.type` - not only for the value `mobile`.

Reply:
I've just applied this suggestion, cheers!